exporter/googlecloudexporter/config.go

```diff
@@ -96,7 +92,7 @@ func createDefaultGCPConfig() *gcp.Config {
 					TargetKey: "location",
 				},
 				{
-					SourceKey: "namespace",
+					SourceKey: "host.name",
 					TargetKey: "namespace",
```

Comment:
These changes to namespace make it unconfigurable; Changing namespace in the config will not change it on exported metrics. I think we do need the 2 separate keys to handle this case.

Reply:
Yeah, I've reverted the change. It is out of scope anyway.